src/App.tsx

```diff
@@ -1,93 +1,136 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { useState, useEffect } from 'react';
+import { HashRouter as Router } from 'react-router-dom';
+
+import { AddTodoForm } from './components/AddTodoForm';
+import { TodoList } from './components/TodoList';
+import { TodosFilter } from './components/TodosFilter';
+import { Todo } from './types/types';
 
 export const App: React.FC = () => {
+  const [inputText, setInputText] = useState('');
```

Comment:
you don't share it with other components except `AddTodoForm` so move this state there

Reply:
++